src/_rules/static.js

```diff
@@ -0,0 +1,171 @@
+import { globalKeywords, handler as h, makeGlobalStaticRules, positionMap } from '#utils';
+
+export const appearances = [
+  ['visible', { visibility: 'visible' }],
+  ['invisible', { visibility: 'hidden' }],
+  ['backface-visible', { 'backface-visibility': 'visible' }],
+  ['backface-hidden', { 'backface-visibility': 'hidden' }],
+  ...makeGlobalStaticRules('backface', 'backface-visibility'),
+];
+
+const cursorValues = ['auto', 'default', 'none', 'context-menu', 'help', 'pointer', 'progress', 'wait', 'cell', 'crosshair', 'text', 'vertical-text', 'alias', 'copy', 'move', 'no-drop', 'not-allowed', 'grab', 'grabbing', 'all-scroll', 'col-resize', 'row-resize', 'n-resize', 'e-resize', 's-resize', 'w-resize', 'ne-resize', 'nw-resize', 'se-resize', 'sw-resize', 'ew-resize', 'ns-resize', 'nesw-resize', 'nwse-resize', 'zoom-in', 'zoom-out'];
+export const cursors = [
+  [/^cursor-(.+)$/, ([, c]) => ({ cursor: h.global(c) })],
+  ...cursorValues.map((v) => [`cursor-${v}`, { cursor: v }]),
+];
+
+export const pointerEvents = [
+  ['pointer-events-auto', { 'pointer-events': 'auto' }],
+  ['pointer-events-none', { 'pointer-events': 'none' }],
+  ...makeGlobalStaticRules('pointer-events'),
+];
+
+export const resizes = [
+  ['resize-x', { resize: 'horizontal' }],
+  ['resize-y', { resize: 'vertical' }],
+  ['resize', { resize: 'both' }],
+  ['resize-none', { resize: 'none' }],
+  ...makeGlobalStaticRules('resize'),
+];
+
+export const userSelects = [
+  ['select-auto', { 'user-select': 'auto' }],
+  ['select-all', { 'user-select': 'all' }],
+  ['select-text', { 'user-select': 'text' }],
+  ['select-none', { 'user-select': 'none' }],
+  ...makeGlobalStaticRules('select', 'user-select'),
+];
+
+export const whitespaces = [
+  [
+    /^whitespace-([-\w]+)$/,
+    ([, v]) => ['normal', 'nowrap', 'pre', 'pre-line', 'pre-wrap', 'break-spaces', ...globalKeywords].includes(v) ? { 'white-space': v } : undefined,
+    { autocomplete: 'whitespace-(normal|nowrap|pre|pre-line|pre-wrap|break-spaces)' },
+  ],
+];
+
+// this is left flexible intentionally
+export const contents = [
+  [/^content-(.+)$/, ([, v]) => ({ content: h.bracket.cssvar(v) })],
+  ['content-empty', { content: '""' }],
+  ['content-none', { content: '""' }],
+];
```

Comment:
I see that in [unocss](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/_rules/static.ts#L76-L82) we also have rules for `content-visibility`. I don't see it in Tailwind but it seems to be used in css in Finn code sometimes. Would that be a reason for us to add those rules?

Reply:
Those are already supported with their TW/Fabric name [here](https://github.com/warp-ds/drive/blob/97973112eadfaf4feefa432467eda1e3a55d2eca/src/_rules/static.js#L3)

Reply:
I asked because it looks like `content-visibility` is a separate CSS property [source](https://developer.mozilla.org/en-US/docs/Web/CSS/content-visibility)

Reply:
That's a very recent spec that isn't well supported.